README.md

```diff
@@ -31,17 +31,36 @@ pip install git+https://github.com/bokulich-lab/RESCRIPt.git
 ```
 
 ### Option 2: Install within QIIME 2 environment
-First activate your QIIME 2 environment (ver 2022.8 or later) and install relevant dependencies:
+First activate your QIIME 2 environment (ver `2023.2` or later) and install relevant dependencies. 
+The typical apporoach will be something like the command below. *Make sure the qiime version within the 
+http string matches the version of the active qiime environment.*
 
 ```
-conda activate qiime2-2023.2
-conda install -c conda-forge -c bioconda -c qiime2 -c https://packages.qiime2.org/qiime2/2023.5/tested/ -c defaults \
+conda activate qiime2-2023.7
+conda install -c conda-forge -c bioconda -c qiime2 -c https://packages.qiime2.org/qiime2/2023.7/tested/ -c defaults \
   xmltodict 'q2-types-genomics>2023.2' ncbi-datasets-pylib
 ```
 Install source:
 
 ```
 pip install git+https://github.com/bokulich-lab/RESCRIPt.git
+
+qiime rescript --help
+```
+
+#### NOTE on installing RESCRIPt within `qiime2-amplicon-2023.9`: 
```

Comment:
I think this will already change in the next release...

Reply:
Yeah, but will this work for 2023.9 though? If not, we'll have to keep this note for that version right?